### ISSUE TITLE
[BUG] H011 false positive

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#44]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3,7
 - djLint Version: 0.4.9

## Issue
The following html, copied from https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#viewport_basics , causes H011

## How To Reproduce
```html
<meta name="viewport" content="width=device-width, initial-scale=1">
```

**Comment by christopherpickering** [comment]
Thanks! The meta tag is kind of unique. I'll split the regex for that rule and give meta its own pattern... w/out width :)
```
<(?:meta)\s+?[^>]*?(?:class|id|src|alt|style|lang|title|href|action|method|name)=[a-zA-Z_-]+
```

**Comment by christopherpickering** [comment]
I suppose that the tag names and attribute names could me made more generic so custom names would fit in.
